styles/index.scss

```diff
@@ -15,13 +15,28 @@ $heading-font: 'Nunito', 'Avenir', sans-serif;
   width: 100%;
 }
 
+.top-photo {
+  padding: 0px 0px 0px 15px;
+  position: absolute;
```

Comment:
i'd suggest instead of explicitly specifying positioning and a width, i would look at the CSS classes that the volunteer page uses to position two thing side-by-side. these use flexbox and automatically shift if there isnt enough space for them. i.e. by putting one side on top of the other rather than squeezing them side-by-side.
![Screenshot_20200629_133903](https://user-images.githubusercontent.com/17362949/86053792-e8b8fc80-ba0d-11ea-8a9c-d4f514bd67ed.png)